### PR TITLE
Add resource prefix and fix integration with AS

### PR DESCRIPTION
**Pull request by Vandalko**
Fixes:
- https://github.com/googlemaps/android-maps-utils/issues/259
- https://github.com/googlemaps/android-maps-utils/issues/252

### CODE REVIEW

**Comment by Vandalko** [comment]
@roman-mazur @Burbaka pls review

**Comment by Burbaka** [comment]
You could make PR to original repo and read an article about it :)

File: .travis.yml [hunk 1]
```diff
@@ -44,4 +44,4 @@ before_script:
   - android-wait-for-emulator
 
 script:
-  - ./gradlew connectedCheck -x library:signArchives -PdisablePreDex
```
Comment (Burbaka) [inline .travis.yml:47]: I thought we are not doing checkstyle changes in such PRs

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Thu Feb 25 11:15:38 AEDT 2016
+#Thu Mar 24 15:46:15 EET 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.11-all.zip
```
Comment (Burbaka) [inline gradle/wrapper/gradle-wrapper.properties]: same as for checkstyle

File: library/build.gradle [hunk 3]
```diff
@@ -23,7 +23,9 @@ dependencies {
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.0"
+    buildToolsVersion "23.0.2"
+
+    resourcePrefix 'amu_'
```
Comment (Burbaka) [inline library/build.gradle:17]: so, as I understand from your change list, this is main fix, isn't it?

**Comment by Burbaka** [comment]
Nevermind, I see that there are already PRs are present.
:+1:

File: .travis.yml (hunk 1 above)
Comment (Vandalko) [inline .travis.yml:47]: it is not checkstyle change - I've edited this file directly on GitHub and it added newline at the end of file automatically

File: library/build.gradle (hunk 3 above)
Comment (Vandalko) [inline library/build.gradle:17]: it's a fix of https://github.com/googlemaps/android-maps-utils/issues/252

**Comment by roman-mazur** [comment]
I see some feedback on your PR to the upstream repo. Let's address comments there.
I would also prefer keeping master branch of this repo in sync with the upstream. You can submit changes that are not merged to the upstream as PRs to dev branch.

So we'll keep master in sync with the upstream, and will be merging master to dev when changes are merged to the upstream.

**Pull request closed without merging**